src/Fixer/FunctionNotation/DateTimeFixer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\FunctionNotation;
```

Comment:
Maybe `ClassUsage` makes more sense than `FunctionNotation`?

Reply:
let us first discuss about idea itself, not implementation details ;)